### PR TITLE
[10.0][FIX] stock_inventory_revaluation: unable to revaluate multiple quants with real method

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**
I got "Expected singleton" error while revaluating multiple quants at the same time

### CODE REVIEW

**Comment by Cedric-Pigeon** [comment]
@mreficent @jbeficent  Could you have a look ? Thanks

**Review by MiquelRForgeFlow — APPROVED** [review]
:+1: 😱

File: stock_inventory_revaluation/models/stock_inventory_revaluation.py [hunk 1]
```diff
@@ -272,8 +272,9 @@ def _create_accounting_entry(self):
             amount_diff = 0.0
             if self.product_id.cost_method == 'real':
                 if self.reval_quant_ids:
```
Comment (pedrobaeza) [inline stock_inventory_revaluation/models/stock_inventory_revaluation.py]: Now you can remove this `if`

Comment (Cedric-Pigeon) [reply stock_inventory_revaluation/models/stock_inventory_revaluation.py]: @pedrobaeza Done, thanks !

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**